### ISSUE TITLE
Use Case: BeachCOMBERS

### ISSUE THREAD

**Issue by AbbyBenson** [tdwg/how-did-it-die#10]
[CenCOOS](https://www.cencoos.org/) has a dataset of volunteer based surveys of marine mammal and seabird carcasses that wash-up on beaches. https://mlml.sjsu.edu/beachcombers/

I believe all records in this dataset would use:
`vitality` property with the vocabulary term "dead"

I'm not sure any of the other associated terms would be able to be implemented (`causeOfDeath`, `underlyingCausesOfDeath`, `deathDate`). The term `vitalityRemarks` if that is a general comments type of field.

cc'ing CenCOOS folks @patcdaniel and @dianalg in case they have anything to add or just want to follow along.

**Comment by gdadade** [comment]
`causeOfDeath` would be "unknown" in this case I assume

**Comment by AbbyBenson** [comment]
I'm honestly not sure we would use the extension for this dataset. I think we would only use the `vitality` core term. The main reason for our interest in this task group is that we want it to be clear to downstream users of this data, especially when integrated with other datasets, that these observations are of dead organisms. We are concerned that the geolocations for these organisms would be confusing for downstream users if this is not made clear.

**Comment by patcdaniel** [comment]
I agree @albenson-usgs. Some of the survey groups try to attribute a cause of death, looking for signs of oiling, DA toxicity, starvation, etc, but those are not necessarily conclusive, more of a co-morbidity. `vitality` is correct in the sense that we are not able to distinguish a seabird that dies on the beach versus one that dies offshore and the carcass washes up onto the beach.
